src/cms/templates/pages/page_view.html

```diff
@@ -2,9 +2,22 @@
 {% load i18n %}
 
 {% block raw_content %}
-<h1>{{page_translation.title}}</h1>
-
-<div style="padding-top: 30px">
-    {{ page_translation.text|safe }}
+<div class="px-4">
+    {% if mirrored_translation and mirrored_page_first %}
+        <h1 class="pt-4">{{mirrored_translation.title}}</h1>
+        <div class="pt-2">
+            {{ mirrored_translation.text|safe }}
+        </div>
+    {% endif %}
+    <h1 class="pt-4">{{page_translation.title}}</h1>
+    <div class="pt-2">
+        {{ page_translation.text|safe }}
+    </div>
+    {% if mirrored_translation and not mirrored_page_first %}
+        <h1 class="pt-4">{{mirrored_translation.title}}</h1>
```

Comment:
Usually we just concatenate the strings of the content. I'm not sure if we want to show the title here. In my opinion we should just add the two content strings in the correct order and display them.

Reply:
Alright, so that means we just want to show the title of the main page and omit the title of the mirrored page. I will adjust that.